src/electron/device-connect-view/components/device-connect-port-entry.tsx

```diff
@@ -52,6 +47,11 @@ export class DeviceConnectPortEntry extends React.Component<DeviceConnectPortEnt
         );
     }
 
+    private onPortTextChanged = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
```

Comment:
Curious to know the reason/thought process for moving the `onPortTextChanged` function as a private member of the `class` rather than a `closure` in the `render` function.

Reply:
Consistency within the file and with the rest of the code base. I guess we do have functions inside functions but is not the general practice, it's more the exception.

Reply:
General practice in our code base. Do we want to even check the pros/cons of that approach? I mean, if we are doing something in a particular way and a new way come along; doesn't it give us an opportunity to analyze the earlier choice? I dont have a problem with this approach; but just stating what I think of this change.

Reply:
Does that sound like something you could bring up on eng discussions?